### ISSUE TITLE
Stuck In Class Selection Room

### ISSUE THREAD

**Issue by Pwn3d12345** [Wynncraft/Issues#4165]
So I haven't played this server for about an year or so. I decided to come back and try it out again. Everything was fine, but I wanted to change my class so I typed /toggle autojoin, then relogged. /class was bugged for some reason and just freezes my game. After choosing a server, I would spawn in a small room made off wool, I would just fall through a chest and fall endlessly into the void. I am unable to do anything or type any commands. I've seen this problem with other people's post, even one from December. I'm assuming that this problem is either being worked on or being ignored, but if any admins could help me, that would be great, as I really wanted to get back into this server. My username is Pwn3d12345.

TL;DR: I am stuck in the class selection room.
